### ISSUE TITLE
Time limit for opened issues

### ISSUE THREAD

**Issue by DDKatch** [datarockets/datarockets-style#42]
I'm not really active here, however I found out a lot of issues 11+ days here are still opened. This styles are vital fo our current project, so I'd like to set up them earlier. I'm not pushing time schedule, but It would be cool to have some time limits for opened issues and some default behavior (usually it's reject as I saw in many repos)

So I propose:
1) to have 4 weeks limit for any issue
2) to have a rejection as default action after 2 weeks of inactivity in comments with closer's comment of course

Anyway you may reopen issue any time, however it will not take some focus, and it will be easier to work on most relevant issues

**Comment by roman-dubrovsky** [comment]
@DDKatch  no, I don't agree. Some issues I'm using as backlog and won't remove them. But if feel that some issue can be closed just ping a creator about it 😉 

> This styles are vital fo our current project, so I'd like to set up them earlier

so feel free to implement some rule and push guys to make discussion and any decisions 😉. Also, we can add any rules for releases (e.g. one week from merged PR to master)

**Comment by DDKatch** [comment]
I don't like the practice to use issues as backlog platform at all. It's about all open source activities. Because it makes much harder for newbie to dig into project's needs(Prioritization issue). It's nice and cool that we have a lot of discussion here and I feel opensource breathing, but it's not spreading from our every project and I'd like just to have a single contributing process everywhere. 

I think with task board there will no such a problem at all, but I don't expect it appears immediately.

@roman-dubrovsky thoughts?

**Comment by roman-dubrovsky** [comment]
@DDKatch yes, but I believe that labels should help us in this case (e.g `postponed` for delayed issues, and `good first issue` and `help wanted` for newcomers. 

But if says a true I didn't see on this problem before. Do you have any other solution so that we won't have much bureaucracy

**Comment by TheBlackArroVV** [comment]
I think, we can care about both variants
We can live some of issues as backlog
And for others add a voting about issue, so everyone can vote and then we can submit the voices and close the issue
Idea about votes was noticed by @AleksSenkou(https://github.com/datarockets/datarockets-style/issues/40#issuecomment-512815203)

**Comment by AleksSenkou** [comment]
Agree that we don't need to do something with opened issues - if you still that some are not matters - right it down

**Comment by roman-dubrovsky** [comment]
As a summary, what do you think about this process:

- we should resolve the issue during the week after his opening
- we have the next steps for processing it:
  1) we can resolve and close it during the week and if it does not need any changes in the codebase (e.g. some questions or duplicated feature requests
  2) if this is a high priority issue or dependency issue, we should add them to the current release milestone and release it in the next version 
  3) if this isn't a high priority issue but we have resources (time and motivated developer how can resolve it) we can add a new issue to the current release milestone 
  4) if we don't have resources for the current milestone, but want to resolve it faster, we can add it to the next release milestone 
  5) if we don't want to resolve some issue now (or just don't have ideas) but want to leave it to future, we should label the issue via `postponed` label. 

cc @AleksSenkou @DDKatch @TheBlackArroVV

**Comment by AleksSenkou** [comment]
> week after his opening

week is a very short period. anyway, time dependence brings management that someone should do and practice shows that it's become broken during some period (especially for opensource project)

We had exactly the same issue with Playbook Keapers team. Finished up with doing nothing with opened issues - someone can go through them (if wants to) and close deprecated, mark postpone those that seem to be postponed, raise additional questions.

**Comment by roman-dubrovsky** [comment]
> week is a very short period

I think this is enough time for moving the issue into some group. Anyway I'll still have an old long-time unclosed issue, but also we'll process fast new issues

**Comment by AleksSenkou** [comment]
@roman-dubrovsky my suggestion is to avoid new processes if we can deal without them (without effectivity lost)

How measuring new issues in a week can help us?

**Comment by roman-dubrovsky** [comment]
@AleksSenkou ok, let's forget about time limits for now (for me that really looks overhead + there are not a good solution for managing long-time issues)

But by this issue, I have a question about minimal time for issues.  E.g this issue #73 has been created, had been discussed only one day, and then we have created a PR #74. Should we added some freezing time so that everybody can check, investigate the issue and leave them thoughts. E.g. not to create PR for the issue before 3 working days from creation (or ping in the slack about issue if you're going to resolve it faster, e.g for critical issues)
